dot/core/service.go

```diff
@@ -579,3 +609,33 @@ func (s *Service) GetMetadata(bhash *common.Hash) ([]byte, error) {
 	s.rt.SetContextStorage(ts)
 	return s.rt.Metadata()
 }
+
+// RegisterRuntimeUpdatedChannel function to register chan that is notified when runtime version changes
+func (s *Service) RegisterRuntimeUpdatedChannel(ch chan<- runtime.Version) (byte, error) {
```

Comment:
Would be nice if we have an Unregister method that closes the channel and remove it from the map

Reply:
added Unregister method.